docker/env/version

```diff
@@ -1 +1 @@
-1.51-pip-23.3
+1.51-argus-0.11.7-update
```

Comment:
Next time update the version on the beginning

Reply:
I think it is better to do this time too, not only next time.

Reply:
Fixed